src/main/java/org/folio/dao/ErrorLogDao.java

```diff
@@ -12,16 +13,26 @@
    * Gets {@link ErrorLog}
    *
    * @param jobExecutionId id of job execution
-   * @param tenantId         tenant id
+   * @param tenantId       tenant id
    * @return future with {@link ErrorLogCollection}
    */
-  Future<ErrorLogCollection> getByJobExecutionId(String jobExecutionId, int offset, int limit, String tenantId);
+  Future<ErrorLogCollection> get(String jobExecutionId, int offset, int limit, String tenantId);
+
+  /**
+   * Gets {@link ErrorLog}
+   *
+   * @param jobExecutionId id of job execution
+   * @param reason         substring of reason in error log
+   * @param tenantId       tenant id
+   * @return future with list of {@link ErrorLog}
+   */
+  Future<List<ErrorLog>> getByJobExecutionIdAndReason(String jobExecutionId, String reason, String tenantId);
```

Comment:
Can we make this more generic to call it getByQuery? So that we can use it for querying by other fields too instead of just reason

Reply:
I think not, cause when we are looking by reason, we check is reason in error logs contain reason from a method. I used 'LIKE' operator, for other fields, we will not need to use 'LIKE'.

Reply:
@KVupp also we have get method, where we can put a query